Automattic/wp-calypso
# Stats: Adjust Stats purchase wizard for WPCOM sites

Related to #80888 

## Proposed Changes

* Tweak the unopened panel title display and button width for all kinds of sites.
* Adjust the purchase wizard styling for WPCOM sites.

## Testing Instructions



* Spin up the change with the Calypso Live link.
* Navigate to Stats > `Traffic` page with the feature flag for a WPCOM site without any purchase or plan: `/stats/day/{site-slug}?flags=stats/paid-wpcom-stats`.
* Click on the `Upgrade my Stats` button of the upsell notice.
* Ensure the purchase wizard display is in line with the design.

<img width="1267" alt="截圖 2023-09-12 上午10 55 34" src="https://github.com/Automattic/wp-calypso/assets/6869813/c381495f-4981-419e-9882-aefbecf18eee">

* Click on the `Personal site` button.
* Ensure the Personal purchase page display is in line with the design.

<img width="1245" alt="截圖 2023-09-12 下午12 04 33" src="https://github.com/Automattic/wp-calypso/assets/6869813/888dd354-be41-4eaf-bf08-00598568d9f4">

* Click on the step 1 button back to the wizard. 
* Click on the `Commercial site` button.
* Ensure the Commercial purchase page display is in line with the design.

<img width="1253" alt="截圖 2023-09-12 下午12 04 40" src="https://github.com/Automattic/wp-calypso/assets/6869813/47e360e0-058e-4bfa-ab9b-83f887f4e2b5">

## Pre-merge Checklist
